browser/ui/toolbar/brave_app_menu_model.cc

```diff
@@ -508,3 +530,31 @@ int BraveAppMenuModel::GetIndexOfBraveSidebarItem() const {
   return sidebar_item_index + 1;
 }
 #endif
+
+int BraveAppMenuModel::GetIndexOfBraveVPNItem() const {
+#if BUILDFLAG(ENABLE_BRAVE_VPN)
+  int vpn_item_index = -1;
+
+  vpn_item_index = GetIndexOfCommandId(IDC_SIDEBAR_SHOW_OPTION_MENU);
```

Comment:
This is kind of a weird way of organizing the items; I realize it's like this everywhere else here too 😄 

But maybe a different approach would be like:
```cpp
std::vector<int> commandsToCheck = {
  IDC_SIDEBAR_SHOW_OPTION_MENU,
  IDC_SHOW_BRAVE_SYNC,
  IDC_MANAGE_EXTENSIONS,
  IDC_SHOW_BRAVE_WALLET,
  IDC_SHOW_DOWNLOADS
};

int vpn_item_index = -1;
for (size_t i = 0; i < commandsToCheck.size(); i++) {
    vpn_item_index = GetIndexOfCommandId(commandsToCheck[i]);
    if (vpn_item_index != -1)
      return vpn_item_index + 1;
    }
}
```

Reply:
Oh, it seems we could make this as a common method!

Reply:
Maybe a macro - because it needs to return from the function it's inside of

The DCHECK can go after still of course